google/cloud/aiplatform/metadata/resource.py

```diff
@@ -189,6 +203,70 @@ def update(
         )
         self._gca_resource = update_gca_resource
 
+    @classmethod
+    def list(
+        cls,
+        filter: Optional[str] = None,
+        metadata_store_id: Optional[str] = "default",
```

Comment:
Optional can be removed.

Reply:
done. Actually does it make any difference between mark one as Optional and not optional but has a default value set?

Reply:
I see. Make sense. I update a couple more locations where metadata_store_id was set to "default" but also marked as Optional.